src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  margin: 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
```

Comment:
It should work with any number (from 0 to 5). Sure, you should display it here with 4 active stars, but it should work with any number. Also, modificator should be in `stars` block like `stars stars--4`. Then use SCSS `for` loop and `:nth-child` selector to apply the correct image to the children of `stars--N` you need

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/e4d30a56-737f-4588-8ef4-7e5f77ff4242)

Reply:
isn't fixed

Reply:
still not fixed